src/common/filters/http-exception.filter.ts

```
@@ -0,0 +1,49 @@
+  HttpException,
+  HttpStatus,
+} from '@nestjs/common';
+import { Request, Response } from 'express';
+import { ErrorCode } from '../types/error.types';
+
+@Catch()
+export class HttpExceptionFilter implements ExceptionFilter {
+  catch(exception: unknown, host: ArgumentsHost) {
+    const ctx = host.switchToHttp();
+    const response = ctx.getResponse<Response>();
+    const request = ctx.getRequest<Request>();
```

ESLint

[error] 'request' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)